storage/innobase/xtrabackup/doc/source/advanced/log_enhancements.rst

```diff
@@ -0,0 +1,70 @@
+.. _logs:
+
+================================================================================
+Improved Log statements
+================================================================================
+
+**Percona XtraBackup** is an open-source command-line utility. Command-line tools have limited interaction with the background operations and the logs provide the progress of an operation or more information about errors.
+
+In earlier versions, the log statement did not have a standard structure. Notice in the following examples the variance in the log statements:
+
+* The backup log statement header has the name of the module, ``xtrabackup``, which generated the statement but no timestamp: 
+
+  .. code-block:: mysql
+    
+     xtrabackup: recognized client arguments: --parallel=4 --target-dir=/home/etc/mybackup/pxb-8/bld/backup --backup=1 --user=root --socket=/home/etc/mybackup/PS-8.0-trunk/bld/mysql-test/var/tmp/mysqld.1.sock --xtrabackup-plugin-dir=/home/etc/mybackup/pxb-8/bld/plugin_output_directory 
```

Comment:
this is how I use it. ie from compile dir. Lets use the same from documentation of  backup

xtrabackup --backup --target-dir=/data/backups/

Reply:
Updated target-dir to match docs.